blog/admin.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib import admin
+from blog.models import Post, Category,Comment  # <-- import Category
+# Put today
+
+admin.site.register(Post)
+admin.site.register(Category)
+admin.site.register(Comment)
+
+# Register your models here.
+# blog/admin.py
+
+class CategoryAdmin(admin.ModelAdmin):
+    pass
```

Comment:
Hi Astrid,

Check the documentation, there are a few more steps to take in order to change the admin interface to show categories as part of a post and remove posts from the categories section.